test/interaction/interaction.test.js

```diff
@@ -394,7 +394,7 @@ describe('resume after interaction', () => {
       await this.agent.get('/auth/resume')
         .expect(200)
         .expect('content-type', 'text/html; charset=utf-8')
-        .expect(/<body onload="javascript:document\.forms\[0]\.submit\(\)"/)
+        .expect(/<body>/)
```

Comment:
remove this line and add the one asserting that a self submitting code is present.